### ISSUE TITLE
Coverage incorrectly reported

### ISSUE THREAD

**Issue by helenst** [DjangoGirls/djangogirls#329]
codecov.io says applications app coverage is 100% but it is not.

It doesn't seem to be including all the files, e.g. here:

https://codecov.io/gh/DjangoGirls/djangogirls/tree/0e6089bd1f05944dc55afb339bab261dcc5b140a/applications

It also lists a test file as having test coverage which doesn't seem right.

**Comment by helenst** [comment]
![screenshot-codecov io 2016-12-06 11-55-01](https://cloud.githubusercontent.com/assets/217966/20924698/d5ca8162-bbaa-11e6-8f65-8da2d303d962.png)

I can't see why it would include `test_forms.py` (which shouldn't be there) but not `models.py` (which should be there).

FYI `pytest --cov` gives the following for the applications app:
```
applications/__init__.py                                       0      0   100%
applications/admin.py                                         62     24    61%
applications/decorators.py                                    16      0   100%
applications/forms.py                                         81     11    86%
applications/models.py                                       194     11    94%
applications/templatetags/__init__.py                          0      0   100%
applications/templatetags/applications_tags.py                12      0   100%
applications/urls.py                                           3      0   100%
applications/utils.py                                         46      6    87%
applications/views.py                                        154     47    69%
```

There are similar problems in the rest of the app.

**Comment by helenst** [comment]
I *think* this is fixed by cf8fc6bbeebbaa36276dff563f844857f7655171 - but for some reason you have to go directly to that commit to see the correct file coverage report: https://codecov.io/gh/DjangoGirls/djangogirls/tree/cf8fc6bbeebbaa36276dff563f844857f7655171 and the default view continues to show incorrect file lists (and references the previous commit ID). Perhaps will be fixed by a commit that actually changes covered files?
